charts/argo-rollouts/templates/controller/deployment.yaml

```diff
@@ -45,6 +45,9 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "argo-rollouts.serviceAccountName" . }}
+      {{- if ne .Values.serviceAccount.automountServiceAccountToken nil }}
+      automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
+      {{- end }}
```

Comment:
I want to follow the existing logics and default value in argo-helm. 😃 
Ref: 
* https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L3087
* https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/argocd-repo-server/deployment.yaml#L75

Reply:
@yu-croco thanks, I wanted to keep default rendered templates as is but it indeed makes sense to set it to true provided that it's the default anyways and other argocd charts use this pattern. Just committed the change.